webapp/client/src/pages/PauschbetragPersonAPage.test.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import PauschbetragPersonAPage from "./PauschbetragPersonAPage";
+import { Default as StepFormDefault } from "../stories/StepForm.stories";
+
+let props = {
+  stepHeader: {
+    title: "Title",
+    intro: "Intro",
+  },
+  form: {
+    ...StepFormDefault.args,
+  },
+  fields: {
+    personAWantsPauschbetrag: {
+      selectedValue: undefined,
+      options: [
+        {
+          value: "yes",
+          displayName: "Ja",
+        },
+        {
+          value: "no",
+          displayName: "Nein",
+        },
+      ],
+      errors: [],
+    },
+  },
+  prevUrl: "/some/prev/path",
+};
+
+describe("With default props", () => {
```

Comment:
Should we check the personAWantsPauschbetrag.selectedValue: "yes" case? (correct radio field selected)

Reply:
And the "no" case of course :)

Reply:
I feel like that is already tested sufficiently for the radio button itself, don't you think?

Reply:
Hm, the integration between the props and the radio component is within the page itself. So your right, the component is sufficiently testet but I would test the integration between the page model and the component itself - you don't have to check if a click is working, but you can validate the model and if the page represents the model correctly.